assets/scss/6-components/navbar/_navbar.scss

```diff
@@ -134,10 +148,17 @@
     padding: $size-b/2 0 0 0;
   }
 
+  // bottom margin is so you can have multiple sets of dropdown items (for example
+  // one with nav links, one with log in/log out buttons) that are spaced a bit
   &__dropdown-items {
     display: flex;
     flex-wrap: wrap;
     list-style: none;
+    margin-bottom: $size-xxxs;
+
+    &:last-of-type {
+      margin-bottom: 0;
+    }
   }
```

Comment:
Just to clarify, this means that in texastribune, the Logout button is moving outside of that `c-navbar__dropdown-items` ul right?

Reply:
It, along with a "Account" button, will live in a separate `c-navbar__dropdown-items` below the first one. I like being able to separate "pure" links from more meta stuff like log out, account, site search, etc.

Reply:
No that makes sense, I was just trying to envision that would transfer over to our tt nav.